sdcm/nemesis.py

```diff
@@ -5076,21 +5079,25 @@ def disrupt_bootstrap_streaming_error(self):
 
         bootstrapabortmanager = NodeBootstrapAbortManager(bootstrap_node=new_node, verification_node=self.target_node)
 
-        bootstrapabortmanager.run_bootstrap_and_abort_with_action(terminate_pattern, abort_action=new_node.stop_scylla)
-        bootstrapabortmanager.clean_and_restart_bootstrap_after_abort()
+        with ignore_stream_mutation_fragments_errors(), ignore_raft_topology_cmd_failing(), ignore_raft_transport_failing():
+            bootstrapabortmanager.run_bootstrap_and_abort_with_action(
+                terminate_pattern, abort_action=new_node.stop_scylla)
+            bootstrapabortmanager.clean_and_restart_bootstrap_after_abort()
 
-        if new_node.db_up() and not self.target_node.raft.is_cluster_topology_consistent():
-            LOGGER.error("New host %s was not properly bootstrapped. Terminate it", new_node.name)
-            bootstrapabortmanager.clean_unbootstrapped_node()
-            self.log.info("Failed bootstrapped node %s removed. Cluster is in initial state", new_node.name)
-            raise BootstrapStreamErrorFailure(f"Node {new_node.name} failed to bootstrap")
+            if new_node.db_up() and not self.target_node.raft.is_cluster_topology_consistent():
+                LOGGER.error("New host %s was not properly bootstrapped. Terminate it", new_node.name)
+                bootstrapabortmanager.clean_unbootstrapped_node()
+                self.log.info("Failed bootstrapped node %s removed. Cluster is in initial state", new_node.name)
+                raise BootstrapStreamErrorFailure(f"Node {new_node.name} failed to bootstrap")
 
         if new_node.db_up() and self.target_node.raft.is_cluster_topology_consistent():
             self.log.info("Wait 5 minutes with new topology")
             time.sleep(300)
-
             self.log.info("Decommission added new node")
-            self.cluster.decommission(new_node, timeout=7200)
+            un_nodes = self.cluster.get_nodes_up_and_normal()
+            with Nemesis.run_nemesis(node_list=un_nodes, nemesis_label="BootstrapStreaminError") as verification_node, \
+                    MonitorDecommissionOperation(target_node=new_node, verification_node=verification_node, timeout=7300):
+                self.cluster.decommission(new_node, timeout=7200)
```

Comment:
if there is a connection between the two timeouts here, code should show,
like:

```python
decomission_timeout=7200
monitoring_timeout=decomission_timeout  + 100
```

since at some point we'll probably will want to use adaptive_timeout on this, like we are doing all around, since those numbers are depended on dataset size and more thing, and can take longer then 2h

Reply:
done